server/routes/contacts/manage/addresses/manageAddressesController.ts

```diff
@@ -28,6 +28,14 @@ export default class ManageAddressesController implements PageHandler {
       address,
       cardTitle: this.getExpiredAddressTitle(address.endDate, address.addressTypeDescription ?? 'Address'),
       mostRelevantAddressLabel: getLabelForAddress(address),
+      cardActions: [
```

Comment:
should'nt this go in the nunjuck template?

Reply:
yeah ideally it would but the way this page and the macro were built made it hard to do that in this case. I plan to refactor some more in the future.